pkg/asset/cluster/tfvars.go

```diff
@@ -365,7 +381,7 @@ func (t *TerraformVariables) Generate(parents asset.Parents) error {
 			installConfig.Config.Platform.OpenStack.OctaviaSupport,
 			string(*rhcosImage),
 			clusterID.InfraID,
-			installConfig.Config.AdditionalTrustBundle,
```

Comment:
This seems a bit shady.  What if the user has an OpenStack with a self-signed cert (so you want to provide `caCert`) and a proxy for some other activity (so you want to provide `AdditionalTrustBundle`)?  I think you need logic above that will generate a single cert blob that is the union of both, or you need to create separate Terraform entries for the two different CA bundles.

Reply:
hmm, good point. Let me double check that.

Reply:
Ah ok, so I double checked. This is a bit of a strange code path that is only on the openstack side of things right now. That value is only used for adding the CA certificate to the bootstrap ignition shim for openstack clusters so that it can authenticate with openstack when using self signed certificates. This doesn't affect the proxy code in any way, so narrowing the scope should not have any negative consequences.

Reply:
Perhaps we need to merge the AdditionalTrustBundle and the CA cert coming from clouds.yaml when passing it to the bootstrap ignition file. We haven't considered the case where the boostrap nodes talks to SSL terminated OpenStack via a proxy if that's a plausible scenario. However, I'd like to treat this as a bug if you don't mind.

Reply:
> However, I'd like to treat this as a bug if you don't mind.

Did something get filed for tracking this, or are we waiting to see if a customer hits it and files a bug?

Reply:
After consideration, I don't think it's a valid scenario.